src/pages/Embeds/Plio.vue

```diff
@@ -177,6 +214,16 @@ export default {
       // emit if an item's visibility has been toggled
       this.$emit("item-toggle", value);
     },
+    itemResponses: {
+      handler(value) {
+        // whenever itemResponses is updated, re-render the markers
+        // and re-calculate the scorecard metrics
+        if (value != undefined && this.player != undefined) {
+          this.showItemMarkersOnSlider(this.player);
```

Comment:
<img width="505" alt="Screenshot 2021-09-24 at 1 15 57 PM" src="https://user-images.githubusercontent.com/13185699/134637769-6ab3592f-1193-4848-8ca2-8f7db157d9d4.png">
why was `calculateScorecardMetrics` removed?

Reply:
It's being called in two instances
Once, when the `createSession` method is called and when the `itemResponses` array is being filled for the first time, `calculateScorecardMetrics` is called once then, and iterates all the itemResponses to calculate the metrics

In the other instance, this is called when the user is doing some action, let's say submitting, skipping etc. 
So there was no need to call it everytime when the itemResponses update